WeatherApp/Model/WeatherModel.swift

```diff
@@ -8,43 +8,71 @@
 
 import Foundation
 
-struct WeatherModel: Codable {
```

Comment:
`WeatherModel` używamy jako model do logiki aplikacji, dlatego nie potrzebujemy `Codable`, bo nie mapujemy na ten typ odpowiedzi z API.

Reply:
`Codable` to tak na prawdę `typealias Codable = Encodable & Decodable`.

`Decodable` to protokół używany dla obiektów, na które chcemy mapować `JSON` z API.
`Encodable` służy do operacji w drugą stronę - jeśli chcemy otrzymać `JSON` z obiektu.